x/programs/runtime/map.go

```diff
@@ -42,14 +41,18 @@ func NewMapModule(log logging.Logger, meter Meter) *MapModule {
 	return &MapModule{
 		meter: meter,
 		log:   log,
-		store: storage{
-			state:   make(map[uint64]maps),
-			mods:    make(map[uint64]api.Module),
-			counter: 0,
-		},
 	}
 }
 
+// GlobalStorage is a global variable that holds the state of all programs.
+// This is a placeholder storage system intended to show how a wasm program would access/modify persistent storage.
+// Needs to be gloabl, so state can be persisted across multiple runtime intances.
+var GlobalStorage = storage{
```

Comment:
Its fine for now but this should not be exported as mutation of these maps is a possible security risk and non thread safe.